tests/dtypes/test_geometry.py

```diff
@@ -135,3 +145,153 @@ def test_geometry_params_srid(
+            'srid': WGS84
+        },
+        Cell(value='POINT (582710 6061887)',
+             link='https://www.openstreetmap.org/?mlat=582710.0&mlon=6061887.0#map=19/582710.0/6061887.0')
+    ),
+    (
+        {
+            'geom_type': 'Point',
+            'geom_value': [582710, 6061887],
+            'srid': LKS94
+        },
+        Cell(value='POINT (25.282777879597916 54.68661318326901)',
+             link='https://www.openstreetmap.org/?mlat=25.282777879597916&mlon=54.68661318326901'
+                  '#map=19/25.282777879597916/54.68661318326901')
+    ),
+    (
+        {
+            'geom_type': 'Polygon',
+            'geom_value': [[[502640, 6035290], [502630, 6035290], [502630, 6035300]]],
+            'srid': WGS84
+        },
+        Cell(value='POLYGON',
+             link='https://www.openstreetmap.org/?mlat=502633.3333333334&mlon=6035293.333333334'
+                  '#map=19/502633.3333333334/6035293.333333334')
+    ),
+    (
+        {
+            'geom_type': 'Polygon',
+            'geom_value': [[[502640, 6035290], [502630, 6035290], [502630, 6035300]]],
+            'srid': LKS94
+        },
+        Cell(value='POLYGON',
+             link='https://www.openstreetmap.org/?mlat=24.040608716271&mlon=54.454444306004426'
+                  '#map=19/24.040608716271/54.454444306004426')
+    ),
+])
+def test_geometry_coordinate_tansformation(rc: RawConfig, value, cell):
+    context, manifest = load_manifest_and_context(rc, f'''
+        d | r | b | m | property                   | type           | ref   | description
+        example                                    |                |       |
+          |   |   | City                           |                |       |
+          |   |   |   | name                       | string         |       |
+          |   |   |   | coordinates                | geometry       |       | WGS(LKS)
+        ''')
+    fmt = Html()
+    dtype = manifest.models['example/City'].properties['coordinates'].dtype
+    geom_func = getattr(geom, value['geom_type'])
+    geometry = geom_func(*value['geom_value'])
+    wkb_element = shape.from_shape(shape=geometry, srid=value['srid'])
+
+    result = commands.prepare_dtype_for_response(
+        context,
+        fmt,
+        dtype,
+        wkb_element,
+        data={},
+        action=Action.GETALL,
+        select=None,
+    )
+    assert result.value == cell.value
+    assert result.link == cell.link
+    assert result.color == cell.color
+
+
+@pytest.mark.parametrize('value, cell', [
+    (
+        {
+            'geom_type': 'Point',
+            'geom_value': [582710, 6061887],
+            'srid': WGS84
+        },
+        Cell(value='POINT (582710 6061887)', link=None)
+    ),
+    (
+        {
+            'geom_type': 'Polygon',
+            'geom_value': [[[502640, 6035290], [502630, 6035290], [502630, 6035300]]],
+            'srid': WGS84
+        },
+        Cell(value='POLYGON', link=None)
```

Comment:
`link` should not be None, it should be a link to OSM map to centroid of polygon.